content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/2-postgresql-db.md

```diff
@@ -138,8 +138,29 @@ Example:
 
 - Database User: `pguser`
 
-```
+```shell
 kubectl get secret app-tarfful-postgresql-3-postgres-pguser-credentials \
- -n app-tarfful-postgresql-3-ns -o jsonpath='{.data.password}' | base64 --decode
+ --namespace app-tarfful-postgresql-3-ns --output jsonpath='{.data.password}' | base64 --decode
+```
+Output:
+```shell
 zFniawyxEVdFtSF9uPfDsjFlOnAeDcrpndi3ReaUbqSGTMSnZ1gawSWkJCLabZR9
-```
\ No newline at end of file
+```
+
+# Next Steps
+
+Add Postgres to your application profile and explore all the capabilities Postgres has to offer. The official Postgres documentation has several [tutorials](https://www.postgresql.org/docs/online-resources/) to help you learn more about Postgres and how to leverage Postgres with your applications.
+
+
+# Resources
```

Comment:
These Resources are more spaced. Our team should probably put our heads together for consistency.